.gitattributes

```diff
@@ -48,3 +48,5 @@
 *.fsproj text=auto
 *.dbproj text=auto
 *.sln text=auto eol=crlf
+
+*.sh eol=lf
```

Comment:
I assume this was intentional, why do we define the EOL for .sh and not for most others?